sorc/build_ufs_coupled.sh

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env bash
```

Comment:
This file can be deleted as we have https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled_sprint/sorc/build_ufs.sh instead

Reply:
Done. This is resolved.